src/main/java/mate/academy/spring/config/WebAppInitializer.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.spring.config;
+
+import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer;
+
+public class WebAppInitializer extends AbstractAnnotationConfigDispatcherServletInitializer {
+    @Override
+    protected Class<?>[] getRootConfigClasses() {
+        return new Class[0];
```

Comment:
Have you run this? And where is your AppConfig?

Reply:
I have AppConfig)) I don't know why it did not push to git